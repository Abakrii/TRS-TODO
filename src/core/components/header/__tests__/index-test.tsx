import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../';

test('renders correctly', () => {
    const tree = renderer.create(<Header headerTitle="test header title" />).toJSON();
    expect(tree).toMatchSnapshot();
});


