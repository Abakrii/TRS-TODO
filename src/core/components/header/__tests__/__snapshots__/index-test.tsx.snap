// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "coral",
      "height": 80,
      "paddingTop": 38,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFF",
        "fontSize": 20,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    test header title
  </Text>
</View>
`;
